src/auth/auth.js

```diff
@@ -245,7 +250,7 @@ router.post('/new-password', async (req, res, next) => {
     var hasura_data = await graphql_client.request(query, {
       username,
       password_hash,
```

Comment:
In other places I think you have replaced this column with just `password`. Here it is `password_hash`. Has to be consistent.